Orange/widgets/visualize/owsieve.py

```diff
@@ -42,6 +42,8 @@ def __init__(self, data, attr1, attr2):
         self.expected = np.outer(self.probs_y, self.probs_x) * self.n
         self.residuals = \
             (self.observed - self.expected) / np.sqrt(self.expected)
+        where_are_NaNs = np.isnan(self.residuals)
+        self.residuals[where_are_NaNs] = 0
```

Comment:
Or you could use

https://docs.scipy.org/doc/numpy/reference/generated/numpy.nan_to_num.html